files/pull-image.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+img=$1
+region=$(echo "${img}" | cut -f4 -d ".")
+ecr_password=$(aws ecr get-login-password --region $region)
```

Comment:
can you add some retries to this? we have a couple reports of rate-limits on `get-login-password`:
- https://github.com/awslabs/amazon-eks-ami/issues/1034
- https://github.com/awslabs/amazon-eks-ami/issues/990#issuecomment-1215628507

Reply:
sure!